src/components/nano/NanoContractHistory.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { t } from 'ttag'
+import ReactLoading from 'react-loading';
+import colors from '../../index.module.scss';
+import hathorLib from '@hathor/wallet-lib';
+import { useDispatch } from 'react-redux';
+import { NANO_CONTRACT_HISTORY_COUNT } from '../../constants';
+import TokenPagination from '../TokenPagination';
+import { reverse } from 'lodash';
+
+
+/**
+ * History of a nano contract
+ *
+ * @memberof Screens
```

Comment:
issue: this is not a @memberof Screens

Reply:
Done, thanks https://github.com/HathorNetwork/hathor-wallet/pull/661/commits/f8840df6edec5eae2fc9669489506bef7c53ae19